### ISSUE TITLE
Assigning PR as REV to the person with 0 reputation

### ISSUE THREAD

**Issue by verkhovin** [zerocracy/farm#1806]
I am a new at zerocracy and **I have no any reputation**.
I applied project [jPeek](https://github.com/yegor256/jpeek) with telegram bot and then recieved the list of messages:

```
Zerocrat, [29.11.18 08:14]
You just got role DEV in the jPeek (https://www.0crat.com/p/C7JGJ00DP) project; you work for free

Zerocrat, [29.11.18 08:14]
You just got role REV in the jPeek (https://www.0crat.com/p/C7JGJ00DP) project; you work for free

Zerocrat, [29.11.18 08:18]
The job #266 (https://github.com/yegor256/jpeek/issues/266) was assigned to you in jPeek (https://www.0crat.com/p/C7JGJ00DP) as REV a minute ago; here is why (https://www.0crat.com/footprint/C7JGJ00DP/108a0d0a-c7d8-4ded-bbcc-4cbbc6acd508);you won't earn any cash on completion

Zerocrat, [29.11.18 08:38]
Your reputation is +0 (below +128); according to §33 (http://www.zerocracy.com/policy.html#33) you can't be a reviewer anymore in jPeek (https://www.0crat.com/p/C7JGJ00DP)

Zerocrat, [29.11.18 08:39]
You just lost role REV in the jPeek (https://www.0crat.com/p/C7JGJ00DP) project

```

So now I have no any reputation and I have `REV` job assigned to me in my agenda. In the same time there is no comments from @0crat in the [issue](https://github.com/yegor256/jpeek/issues/266) about my assignition. Seems like a bug.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@verkhovin[/z](https://www.0crat.com/u/verkhovin) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@verkhovin it's a bug: we should not assign REV role in such case (if it can be unassigned because of low reputation), we should check reputation before assigning the role.

**Comment by 0crat** [comment]
Job [#1806](https://github.com/zerocracy/farm/issues/1806) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @verkhovin[/z](https://www.0crat.com/u/verkhovin)

**Comment by g4s8** [comment]
@verkhovin no, this task is assigned to you, it's not your mistake, you'll be payed on close. But you can refuse any job if you want.

**Comment by vassilevsky** [comment]
This weird sequence of notifications hit me too.
